bindings/node/test/clientEncryption.test.js

```diff
@@ -695,6 +692,142 @@ describe('ClientEncryption', function () {
     });
   });
 
+  describe('encrypt()', function () {
+    let clientEncryption;
+    let completeOptions;
+    let dataKey;
+
+    beforeEach(async function () {
+      if (requirements.SKIP_LIVE_TESTS) {
+        this.currentTest.skipReason = `requirements.SKIP_LIVE_TESTS=${requirements.SKIP_LIVE_TESTS}`;
+        this.test.skip();
+        return;
+      }
+
+      await setup();
+      clientEncryption = new ClientEncryption(client, {
+        keyVaultNamespace: 'client.encryption',
+        kmsProviders: { local: { key: Buffer.alloc(96) } }
+      });
+
+      dataKey = await clientEncryption.createDataKey('local', {
+        name: 'local',
+        kmsProviders: { local: { key: Buffer.alloc(96) } }
+      });
+
+      completeOptions = {
+        algorithm: 'RangePreview',
+        contentionFactor: 0,
+        rangeOptions: {
+          sparsity: new BSON.Long(1)
+        },
+        keyId: dataKey
+      };
+    });
+
+    afterEach(() => teardown());
+
+    context('when expressionMode is incorrectly provided as an argument', function () {
+      it('overrides the provided option with the correct value for expression mode', async function () {
+        const optionsWithExpressionMode = { ...completeOptions, expressionMode: true };
+        const result = await clientEncryption.encrypt(
+          new mongodb.Long(0),
+          optionsWithExpressionMode
+        );
+
+        expect(result).to.be.instanceof(Binary);
+      });
+    });
+  });
+
+  describe('encryptExpression()', function () {
+    let clientEncryption;
+    let completeOptions;
+    let dataKey;
+    const expression = {
+      $and: [{ someField: { $gt: 1 } }]
+    };
+
+    beforeEach(async function () {
+      if (requirements.SKIP_LIVE_TESTS) {
+        this.currentTest.skipReason = `requirements.SKIP_LIVE_TESTS=${requirements.SKIP_LIVE_TESTS}`;
+        this.test.skip();
+        return;
+      }
+
+      await setup();
+      clientEncryption = new ClientEncryption(client, {
+        keyVaultNamespace: 'client.encryption',
+        kmsProviders: { local: { key: Buffer.alloc(96) } }
+      });
+
+      dataKey = await clientEncryption.createDataKey('local', {
+        name: 'local',
+        kmsProviders: { local: { key: Buffer.alloc(96) } }
+      });
+
+      completeOptions = {
+        algorithm: 'RangePreview',
+        queryType: 'rangePreview',
+        contentionFactor: 0,
+        rangeOptions: {
+          sparsity: new BSON.Long(1)
+        },
+        keyId: dataKey
+      };
+    });
+
+    afterEach(() => teardown());
+
+    it('throws if rangeOptions is not provided', async function () {
+      delete completeOptions['rangeOptions'];
```

Comment:
```suggestion
expect(delete completeOptions.rangeOptions).to.be.true;
```

Along with the other deletes it could be confusing if something prevents these from being removed down the road